src/components/Form/Stories/BasicForm.js

```diff
@@ -90,5 +102,10 @@ export const getBasicFormKnobs = () => ({
   ]),
   bsSize: select('Size', [null, 'small', 'large']),
   showHelp: boolean('Show Help', true),
-  disabled: boolean('Disabled', false)
+  disabled: boolean('Disabled', false),
+  content: text(
+    'FieldLevelHelp Content',
```

Comment:
minor nit here: `Field Level Help Content`

Reply:
Addressed in f5628db87128706c1c309208f877c606cf4ae74f